app/services/order_details/param_updater.rb

```diff
@@ -70,7 +70,9 @@ def update_attributes(params)
 
     @order_detail.transaction do
       assign_price_changed_by_user
-      @order_detail.reservation.save_as_user(@editing_user) if @order_detail.reservation
+      if @order_detail.reservation
+        log_and_rollback unless @order_detail.reservation.save_as_user(@editing_user)
```

Comment:
Would it be worth adding a comment here? Or do you think a `git blame` will be enough for future us.

Reply:
Can you think of any scenario where a rollback here would cause problems?  This is going to run every time an order details is updated via `OrderManagement::OrderDetailsController#update` so it will no longer be possible to update an order detail if the reservation is invalid.

Reply:
I'm not sure. There's probably a situation where it's a problem, but if the reservation is invalid then it seems we have another problem. I'm inclined towards leaving the rollback in--assuming it presents some error notification to the user--because then the user will know that the updates to the reservation times didn't work.

And hopefully, we'll see the notification and get enough info from it to finally figure out what's going wrong with these reservations. Having the user aware there was an issue could provide us additional insight as well.